Add project prioritization recipe (#385)
FAIRplus/the-fair-cookbook: content/recipes/introduction/priorization.md

(fcb-prioritization)=
# Prioritization of projects for FAIRification

+++
<br/>

````{panels_fairplus}
:identifier_text: FCB055
:identifier_link: 'https://w3id.org/faircookbook/FCB055'
:difficulty_level: 2
:recipe_type: guidance
:reading_time_minutes: 10
:intended_audience:   
:has_executable_code: nope
:recipe_name: Prioritization of projects for FAIRification
```` 

## Main Objectives 

The main purpose of this recipe is:

> To provide an overview of elements that could help users in prioritisation and selection of potential projects, processes, or data for FAIRification. For each of these elements, we provide comparative benefits between different stages and justify the order of the stages required for FAIRification. Finally, we also highlight the reasons for making the data FAIR.

---
## Introduction

> With the increased awareness in the FAIR principles, the drive to implement them can be felt in projects and programs. However, considering the volume and variety of such projects and the finite nature of available resources, it is also necessary to establish a principle approach to prioritizing project or data FAIRification. The present recipe aims to provide insights into how to go about this by outlining essential elements which have been considered and �battle tested�.  

Reviewer: ```suggestion
With the increased awareness of the FAIR principles, the drive to implement them can be felt in projects and programs. However, considering the volume and variety of such projects and the finite nature of available resources, it is also necessary to establish a *principled approach* to prioritizing datasets for data FAIRification. The present recipe aims to provide insights into how to go about this process by showcasing essential criterian which have been considered and �battle-tested�.  
```

> In the recipe, is structured in the following way:

Reviewer: ```suggestion
The is structured in the following way:
```

> * We would begin by pointing out the reasons why one needs to make their dataset FAIR. 

Reviewer: ```suggestion
* We begin by pointing out the reasons why one needs to make their dataset FAIR. 
```

> * Following this, a brief overview of the dataset corpus formulation. Once the team has collected all the necessary data, it can now pass through the prioritisation phase. Here a thorough assessment of the dataset based on certain criteria is done. This assessment in-turn leads to development of a scorecard.

Reviewer: ```suggestion
* Following this, a brief overview of the dataset corpus formulation. Once the team has collected all the necessary data, it can now pass through the prioritization phase. Here, a thorough assessment of the dataset based on certain criteria is done. This assessment, in turn, leads to the development of a scorecard.
```

> * Finally, with the points each dataset has achieved in the scorecard, a raking can be established and projects or datasets can be prioritized.

Reviewer: ```suggestion
* Finally, with the points each dataset has earned in the scorecard, a ranking can be established and projects or datasets can be prioritized.
```

	- It helps in increasing visibility and impact of the data within the scientific community by standardising the published project/dataset metadata using a well-accepted metadata model and getting indexed by major community resources.
* Data harmonisation:
	- When creating a tool with the help of multiple independent resources, FAIRification can help in easing the integration of data sources, for instance with the help of a common terminology.
* Encourages community collaboration:
	- The FAIRification process can attract communities together for potential collaboration as each of the independent communities would now have a structured data format to map their data for integration.
* Prepares data for downstream tasks:
	- A FAIRified data is machine readable and hence can potentially be used by software agents such as machine learning (ML) or artificial intelligence (AI) models for analysis with help of downstream.
* Adds economic aspect of capitalization on research investment:
	- A FAIRified data is reusable, meaning that initial investment and newly generated data assets are protected but also that additional investment in redoing research can be avoided.

> Additional  reasons for the need of FAIRification, encompassing the ethical reasons for doing so. They are more fully described in a dedicated recipe called [Values of FAIR](https://fairplus.github.io/the-fair-cookbook/content/recipes/introduction/FAIRplus-values.html).

Reviewer: ```suggestion
Additional reasons for the need of FAIRification encompass ethical reasons for doing so. These are more fully described in a dedicated recipe called [Values of FAIR](https://fairplus.github.io/the-fair-cookbook/content/recipes/introduction/FAIRplus-values.html).
```

- A project is in early stages of development when the data requirements for the given project are being listed and simultaneously being collected. At this stage, the data availability is the lowest, and it is easier to design a data model, choose ontologies etc. that ensures creation of FAIR data by design'.
- In the middle stage, all the data relevant for the given project has been collected and now needs to be standardised for downstream tasks such as for predictions using machine learning approaches. This is a stage where there is maximum availability of data. 
- Lastly, in the end stage, data has either been deposited or handed over to the respective heads and the project is near its termination. Furthermore, it is at this stage that the risk  of organisation restructuring is highest, with staff reassignment or departure. Hence, starting a FAIRification process at a project  end stage could be the least favourable as this could place large demands in time and resources on key personnel, such as for corresponding data owners and data generators, at a time of effort wind down.

> **Hence, the best time of engaging a project into the FAIR pipeline would be dependent on the retrospective and prospective aspect of the project.** The best time to FAIRify prospective projects is the early stage. This stage is the best time to define and lay out metadata standards that need to be followed during the project trajectory. As a result, it eliminates many of the downstream logistical and financial problems. On the other hand, the FAIRification of retrospective projects is in the middle stage. This is because it is a data rich stage and contact with both data owners and generators could be established for better understanding of the data.

Reviewer: ```suggestion
**Hence, the best time of engaging a project into the FAIR pipeline would be dependent on the retrospective and prospective aspects of the project.** `The best time to FAIRify prospective projects is the early stage`. This stage is the best time to define and layout metadata standards that need to be followed during the project's trajectory. As a result, it eliminates many of the downstream logistical and financial problems. On the other hand, the FAIRification of retrospective projects is in the middle stage. This is because it is a data-rich stage and contact with both data owners and generators could be established for a better understanding of the data.
```

#### 3. Prioritisation based on partners
	
A project has the potential to involve a large number of people each coming from various institutions. Consequently, users can also consider the wider consortia involved as a criteria for prioritising projects from a list of projects. In the end, the FAIRification process should benefit as many people as possible.

> **Thus, projects that have diverse partners involved (e.g.  academic, industrial, start-up, and so on)  should be prioritised over singleton partners, that is those projects that involve people from the same institute or industrial group.** The main reason for this priority is the impact of the FAIRification process. Within an intra-organization group, the data generators, maintainers, and depositors might have the same terminology and definitions related to the data and hence, a FAIRification process would only be needed when the data has to be deposited on public repositories or databases. On the other hand, in an inter-organization group definitions and terminologies are not consistent, and the transfer of data from data generators to depositors might be time-consuming. As a result, the FAIRification process would benefit by increasing transparency between intra-organizational groups.

Reviewer: ```suggestion
**Thus, projects that have diverse partners involved (e.g.  academic, industrial, start-up, and so on)  should be prioritised over singleton partners, that is those projects that involve people from the same institute or industrial group.** The main reason for this priority is the impact of the FAIRification process. Within an intra-organization group, the data generators, maintainers, and depositors might have the same terminology and definitions related to the data and hence, a FAIRification process would only be needed when the data has to be deposited on public repositories or databases. On the other hand, in an inter-organization group, definitions and terminologies aren't likely to be consistent, and the transfer of data from data generators to depositors might be time-consuming. As a result, the FAIRification process would be beneficial by increasing transparency between intra-organizational groups.
```

#### 4. Prioritisation based on existence of Data Management Plan (DMP)

> A data management plan (DMP) is a document that describes the life cycle of a data beginning from its generation, followed by processing and collection, then dissemination, and finally the usage (More details in [Data Management Plan]() recipe). This established document gives the data owners and FAIR experts an overview of the resource under study. On the basis of this DMP document, the FAIRification process can be determined and established. In case of absence of the DMP, the FAIR experts lack the ability to find the best FAIRification process for the given data thereby leading to inefficient results. Overall, the DMP plan would potentially point to the data regulatory aspect considering the data usage as well as dessiementation of the data into public resources. **Thus, projects that have their DMP documents established should be prioritised over those projects that do not have one ready yet.**

Reviewer: ```suggestion
A data management plan (DMP) is a document that describes the life cycle of a data beginning from its generation, followed by processing and collection, then dissemination, and finally the usage (More details in [Data Management Plan]() recipe). This established document gives the data owners and FAIR experts an overview of the resource under study. On the basis of this DMP document, a FAIRification process can be determined and established. In case of the absence of a DMP, the FAIR experts lack the ability to find the best FAIRification process for the given data thereby leading to inefficient results. Overall, the DMP plan would potentially point to the data regulatory aspect, considering the data usage as well as dissemination of the data into public resources. **Thus, projects that have their DMP documents established should be prioritized over those projects that do not have one ready yet.**
```

**In summary, projects that FAIR experts can get access to data should be at a higher priority than those that have limitations on data access.**

#### 6. Prioritisation based on presence of data champions

**Data champions** can be defined as a group of people that have an expertise in dealing with certain processes, data, or pipeline. When dealing with specific processes or data types, it is beneficial if an expert who has experience in dealing with such processes or data already exists within the FAIR expert team.

**Prioritisation of projects with consideration of expertise required in dealing with FAIRification tasks should be done.** Such projects can then have a predetermined timeline involved, making the step-by-step approach for accomplishing a FAIR process or data comparatively easy.

> #### 7. Prioritisation based on data types
> [comment]: <TODO!!>
> Each data can be available in a number of different formats like sequencing data can be in INSDC SRA format (some cookbook recipe?), or imaging data in DICOM format and more examples here. 
> - Community approved data type format (FAIRSharing)
> - Is there any repository accepting this data type (Array Express)
> 
> 
> 

Reviewer: ```suggestion
#### 7. Prioritisation based on data types

Each project may produce a range of data that may be available in a number of different formats. 
For instance,  sequencing data can be in distributed in FASTQ format while associated metadata be described using INSDC SRA format or imaging data in DICOM format. 
It is therefore possible to rank projects based on this information and more precisely, one may use a resources such as [FAIRsharing](https://fairsharing.org) to do two things:
i. identify the community approved data type format and terminologies supporting a data type
ii. identify a funder or publisher recommended repository accepting this data type (e.g. EMBL EBI Array Express for Transcriptomics data)

```

> Along with the above criteria, two more factors play a crucial role in the prioritisation process. The first being the **time management**. Since the FAIRification process can depend on a number of factors, it is important to acknowledge the time required for the process to complete. The second being **risk management**, with the assessment of potential roadblocks that could interfere essential. Such roadblocks should be identified and rectified if possible prior to starting a FAIRification process. 

Reviewer: ```suggestion
Along with the above criteria, two more factors play a crucial role in the prioritisation process:

- [ ]  The first is the **time management**. Since the FAIRification process can depend on a number of factors, it is important to acknowledge the time required for the process to complete. 

- [ ] The second is  **risk management**, with the assessment of potential roadblocks that could interfere essential. Such roadblocks should be identified and rectified if possible prior to starting a FAIRification process. 

```

| Value criteria | Cost criteria |
| ----------- | ----------- |
| Project focus | Data availability and access |
| Data champion  | Availability of DMP |
| Data Type   | Partners involved |

---
## Scorecard formulation and ranking

Overall, for each criteria, a rank can be assigned based on the priority decided by the PSP team. This ranked-based assessment of projects would in turn lead to development of a scoreboard (Figure/Table). To enable ranking of projects, an additive sum of each of these criteria should be used, thereby directing each project with one final score. In the end, a descending ranking of the projects can be achieved, and the top k number of projects can be selected for FAIRification by the PSP team and handed over to the FAIR experts.

> [comment]: <TODO!! Need help on this part.>

Reviewer: ```suggestion

```

<img></img>

The detailed description of each of the scores can be found on the [scorecard template](https://zenodo.org/record/5778431#.YbikDxNKhH1).

---
## Conclusion

> > Faced with a larger number of projects needing FAIRification, it becomes necessary to establish a process for ranking and prioritizing these projects. The recipe, besides reminding  the benefits of making their data FAIR, provides suggestions for assisting in establishing a prioritization procedure. Hence, the aim of the recipe is to provide the readers with a wider perspective of criteria they could use for ordering projects or datasets for FAIRification and enable them to personalize the ranking of factors based on their needs or requirements. Additionally, it provides the reader a scorecard template that can be translated and used for their use cases.

Reviewer: ```suggestion
> Faced with a larger number of projects needing FAIRification, it is necessary to establish a process for ranking and prioritizing these projects. The recipe, besides reminding the benefits of making their data FAIR, provided suggestions for assisting in establishing a prioritization procedure. Hence, the aim of the recipe was to provide the readers with a wider perspective of criteria they could use for ordering projects or datasets for FAIRification and enable them to personalize the ranking of factors based on their needs or requirements. Additionally, it provides the readers with a scorecard template that may be translated and used for their use cases.
```

> 
> ## What to read next section
> * [Data catalog](fcb-find-bs-catalog)
> * [The Value of FAIR](fcb-intro-fair-values)

> > * Data Management Plan

Reviewer: ```suggestion
> * [Data Management Plan](https://rdmkit.elixir-europe.org/data_management_plan.html)
```

> > * Pointer to RDMtk
> > * AIR+Q Pistoia Alliance paper
> 

Reviewer: ```suggestion

```

(fcb-prioritization)=
# Prioritization of projects for FAIRification

+++
<br/>

````{panels_fairplus}
:identifier_text: FCB055
:identifier_link: 'https://w3id.org/faircookbook/FCB055'
:difficulty_level: 2
:recipe_type: guidance
:reading_time_minutes: 10

> :intended_audience:   

Reviewer: :intended_audience: everyone
